src/Tizen.NUI/src/public/Animation/TransitionOptions.cs

```diff
@@ -97,20 +105,24 @@ private void EnableProvider()
         }
 
         /// <summary>
-        /// Gets or sets the Shared object Id
+        /// During the Page transition, if two Views each of on the old top Page
+        /// and new top Page have same TransitionTag, the View on the old top Page
+        /// will be transition to the one of new top Page.
         /// </summary>
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public String SharedId
-        {
-            set
-            {
-                sharedId = value;
-            }
-            get
-            {
-                return sharedId;
-            }
-        }
+        public string TransitionTag { set; get; } = null;
+
+        /// <summary>
+        /// A View could be transition with its child Views or without them.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool TransitionWithChild { set; get; } = false;
+
+        /// <summary>
+        /// A View could be transition with its child Views or without them.
```

Comment:
Please specify the default value of this property in xml documentation. (true/false)